vlsi/Makefile

```diff
@@ -124,4 +231,4 @@ $(OBJ_DIR)/hammer.d: $(HAMMER_D_DEPS)
 #########################################################################################
 .PHONY: clean
 clean:
-	rm -rf $(VLSI_OBJ_DIR) hammer-vlsi*.log __pycache__ output.json $(GENERATED_CONFS) $(gen_dir)
+	rm -rf $(VLSI_OBJ_DIR) hammer-vlsi*.log __pycache__ output.json $(GENERATED_CONFS) $(gen_dir) $(SIM_CONF) $(SIM_DEBUG_CONF)
```

Comment:
Delete the `SIM_TIMING_CONF` also? What about the `<SYN/POWER>_CONF`s also?

Reply:
Good catch.

Reply:
Fixed. SYN_CONF is included in generated confs. I think that could use a clean up later. There is a makefile change in hammer that will probably change how we want to handle that.